supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/api/request/BatchRequestExecutor.scala

```diff
@@ -188,19 +188,15 @@ object BatchRequestExecutor {
         val time = java.time.Instant
           .ofEpochSecond(e.getEventTime.getSeconds, e.getEventTime.getNanos.toLong)
           .atOffset(java.time.ZoneOffset.UTC)
-        val eventType = e.getDescription match {
-          case startedRegex() => CallMetadataKeys.VmStartTime
-          case endedRegex() => CallMetadataKeys.VmEndTime
-          case _ => e.getType
```

Comment:
Note that here we are calling `getType`, so events that don't correspond to VM start or end time get labeled with the type.